units/crons.go

```diff
@@ -29,7 +29,9 @@ const (
 	TSFormat = "2006-01-02.15-04-05"
 
 	// createHostQueueGroup is the queue group for the provisioning-create-host job.
-	createHostQueueGroup            = "service.host.create"
+	createHostQueueGroup = "service.host.create"
+	// terminateHostQueueGroup is the queue group for host-termination-jobs created by cron jobs.
```

Comment:
All the host termination enqueues have to go into the queue group, not just the ones that are populated by certain jobs. Scopes are queue-specific, so you could enqueue multiple host termination jobs in the different queues, which breaks the point of the lock and could cause correctness problems when two host termination jobs run concurrently.

I would update the comment to not specify the cron job bit, and also ensure [this job](https://github.com/evergreen-ci/evergreen/blob/95ff59c975ee709d63ee25eef592933ea251352c/api/host.go#L113-L119) goes in the new queue.

Reply:
Ah good to know, thanks!